duckduckgo/iOS
# accessibility improvements

Task/Issue URL: https://app.asana.com/0/392891325557410/1114078510432707
Tech Design URL:
CC:

**Description**:

Improve accessibility for the app.  I will reach out to an interested user who will evaluate the app for us and give us some feedback directly.

General concepts:

* Anything that's an action should have the "button" trait. During voice over Siri will say "button" after the item description. 

* Images that don't add any extra info should be ignored

* Don't worry about what's on web pages - that's up to the web pages (though feel free to report accessibility problems on the SERP directly on Asana)

**Steps to test this PR**:
1. Use Siri to turn on Voice Over (note, use Siri to turn it off again, it's the easiest way)
1. Check that all buttons on the home screen are accessible (even if disabled)
1. Check all visible buttons in the omnibar are accessible
1. Change the home screen type and ensure that the center search is accessible
1. Browse to a page and check that buttons on a page are accessible
1. Open the tab switcher and check that it's possible to select tab and close a tab
1. Open bookmarks and check that it's possible to select a bookmark
1. Open settings
1. Open feedback from the settings

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
